.gitignore

```diff
@@ -9,16 +12,19 @@ logs/*
 config/options
 config/options.*
 config/modules.config.js
+config/runtime.config.js
+
+
 
 .DS_Store
 node_modules
 node_modules/*
 tests/config/env
-package-lock.json
```

Comment:
❤️

Reply:
So I obviously didn't change these config files, crazy how that might have happened

Reply:
Hm, you didn't? I like those. Wondering were all these changes are coming form then. Anyways ;)